meshtastic/interfaces/ble/notifications.py

```
@@ -1,19 +1,43 @@
 """BLE notification management."""
 
+import contextlib
 import logging
+import re
+import struct
 from threading import RLock
-from typing import TYPE_CHECKING, Any, Callable
+from typing import TYPE_CHECKING, Any, Callable, cast
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)